Database/Corrections/sodQuestFixes.lua

```diff
@@ -2605,6 +2605,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -425216,
             [questKeys.zoneOrSort] = sortKeys.PRIEST,
         },
+        [90157] = {
+            [questKeys.name] = "Void Plague",
+            [questKeys.startedBy] = {nil,{407734}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 8,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.PRIEST,
+            [questKeys.objectivesText] = {"Find the Gnarlpine Cache at the bottom of the Ben'ethil Barrow Den and loot if for the rune."},
```

Comment:
```suggestion
            [questKeys.objectivesText] = {"Find a Gnarlpine Cache at the bottom of the Ben'ethil Barrow Den and loot if for the rune."},
```

Reply:
...and loot _it_ not _if_ for the rune

Reply:
This has been fixed in: https://github.com/Questie/Questie/pull/5482/commits/bf2691a373292602e9a8e370d90aa8013ee8cddb